File: algorithms/Array/27-RemoveElement/go/remove_ele.go
func removeElement(nums []int, val int) int {
    s_idx := 0
    for q_idx:=0; q_idx<len(nums); q_idx++{
        if nums[q_idx] != val {
            nums[s_idx] = nums[q_idx]
            s_idx ++
        }
    }
    return s_idx
}

File: algorithms/Array/27-RemoveElement/py/remove_ele.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        s_idx = 0
        for q_idx in range(0, len(nums)):
            if nums[q_idx] != val:
                nums[s_idx] = nums[q_idx]
                s_idx += 1
        return s_idx
